packages/minimongo/minimongo_tests.js

```diff
@@ -2099,11 +2103,11 @@ Tinytest.add("minimongo - array sort", function (test) {
   var testCursorMatchesField = function (cursor, field) {
     var fieldValues = [];
     c.find().forEach(function (doc) {
-      if (_.has(doc, field))
+      if (doc.hasOwnProperty(field))
         fieldValues.push(doc[field]);
     });
-    test.equal(_.pluck(cursor.fetch(), field),
-               _.range(_.max(fieldValues) + 1));
+    test.equal(cursor.fetch().map(function (doc) { return doc[field]; }),
+               Array.from({length: Math.max.apply(null, fieldValues) + 1}, function (_, i) { return i; }));
```

Comment:
Same variable name comment as above (maybe use a different name since Underscore is available in this test file).